modules/ref-dashboard-configuration-options.adoc

```diff
@@ -67,9 +75,16 @@ On the *Deploy model* dialog when using the single-model serving platform:
 
 To hide these deployment-mode lists and set the deployment mode to *Advanced* when using the single-model serving platform, set the `disableKServeRaw` value to `true`.
 | `dashboardConfig:
+disableModelCatalog` (Technology Preview) | `true` | Hides the the *Models → Model catalog* option in the dashboard navigation menu. 
+
+This configuration option is not included in the default `OdhDashboardConfig` CR file.
+To show the *Models → Model catalog* option in the dashboard navigation menu, edit the `OdhDashboardConfig` CR `spec:dashboardConfig` section to add the `disableModelCatalog:false` entry.
+
+The model catalog is a Technology Preview feature in this {productname-short} release.
+| `dashboardConfig:
 disableModelMesh` | `false` | Enables the ability to select ModelMesh as a model-serving platform. To disable this ability, set the value to `true`.
 | `dashboardConfig:
-disableModelRegistry` | `false` | Shows the *Model registry* option and the *Settings → Model registry settings* option in the dashboard navigation menu. To hide these menu options, set the value to `true`.
+disableModelRegistry` | `false` | Shows the *Models → Model registry* option and the *Settings → Model registry settings* option in the dashboard navigation menu. To hide these menu options, set the value to `true`.
```

Comment:
Model registry entry looks /lgtm but model registry is also tech preview for summit AFAIR

Reply:
Thank you @manaswinidas , I added the TP text to the MR entries.